src/Components/SideNav/index.scss

```diff
@@ -7,7 +7,7 @@ nav {
   background-color: $white;
   //   border-radius: 0.3rem;
   box-shadow: 0.2rem 0.2rem 0.2rem 0.2rem $light-gray;
-  position: absolute;
+  // position: absolute;
```

Comment:
you can remove the comment here, if its not required.

Reply:
sure